apps/webapp/src/components/shared/input-token.tsx

```diff
@@ -1,10 +1,23 @@
 import { Input, InputProps } from '@penumbra-zone/ui';
 import { cn } from '@penumbra-zone/ui/lib/utils';
-import { displayAmount, fromBaseUnitAmount } from '@penumbra-zone/types';
+import { fromBaseUnitAmount, joinLoHiAmount } from '@penumbra-zone/types';
 import SelectTokenModal from './select-token-modal';
 import { Validation, validationResult } from './validation-result';
-import { AccountBalance } from '../../fetchers/balances';
+import { AccountBalance, AssetBalance } from '../../fetchers/balances';
 import { Selection } from '../../state/types';
+import { Fee } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/fee/v1alpha1/fee_pb';
+
+const PENUMBRA_FEE_DENOMINATOR = 1000;
```

Comment:
Note that we're currently hardcoding the penumbra fee denominator. @Valentine1898 pointed out that I could call [`getAllAssets`](https://github.com/penumbra-zone/web/blob/85fe38e4814b2600214f959a843f932e8004e217/apps/webapp/src/fetchers/assets.ts#L5) to get the denominator of the token referenced via the fee's `assetId` property; but @hdevalence, will there ever be a case where the fee will be paid in something other than the staking token (penumbra)?

Reply:
i believe currently tx fees are always PEN https://protocol.penumbra.zone/main/stake/validator-rewards.html

Reply:
https://discord.com/channels/824484045370818580/979785365437165658/1194805904311328808 
We can ignore the `assetId` for now